install/kubernetes/Makefile

```diff
@@ -17,9 +17,11 @@ CILIUM_CHART_REGEX := '\([vV]ersion:\) '$(VERSION_REGEX)
 CILIUM_TAG_REGEX := '\(tag:\) \(v'$(VERSION_REGEX)'\|latest\)'
 CILIUM_PULLPOLICY_REGEX := '\(pullPolicy:\) .*'
 
+.DEFAULT_GOAL := $(QUICK_INSTALL)
+
```

Comment:
Is there a particular reason / concern you have with not having `all` as the default target?

I'm thinking that once we do backports for older branches where we may change the `/VERSION` file, keeping the `update-versions` step will become important to ensure we properly update the quick-install YAML with the correct cilium-agent versions as well.

Reply:
There is no particular reason, it's just that I am not fully aware of cilium's releasing process for helm charts, so didn't want to either do any version check or update helm version. However, I just check the logic for _update-versions_, it just takes _/VERSION_ value and update helm files, so agree to run it as part of github action.

Hence, I will remove above default target, as it was introduced to avoid calling $(QUICK_INSTALL) path. I will just call `make all` in github action.